lldb/include/lldb/API/SBDebugger.h

```diff
@@ -46,7 +46,8 @@ class LLDB_API SBDebugger {
       eBroadcastBitProgress = (1 << 0),
       eBroadcastBitWarning = (1 << 1),
       eBroadcastBitError = (1 << 2),
-      eBroadcastBitProgressCategory = (1 << 3),
+      // This needs to match the corresponding bit in Debugger.h
```

Comment:
Doesn't this comment apply to the whole enum? My suggestion was to move this above line 45 to convey that, unless that's not the case?

Reply:
My mistake, I was indicating that the progress category bit is skipping `(1 << 3)` in this sequence since that bit has to match the one in `Debugger.h`. I can change the comment to say that the whole enum needs to match `Debugger.h`.